api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -622,8 +622,15 @@ public static final Collection<String> AUTO_ROLES() {
 	/**
 	 * Global property that stores the base url for the application.
 	 */
+	@Deprecated
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation

Reply:
@dkayiwa I reformated it according to the convention.